index.php

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Movie</title>
+</head>
+<body>
+    <div class="homebar">
+        <h1 class="homepage">Welcome To My Movie Database</h1>
+</div>
+<div class="navbar">
+        <a class="home" href="http://localhost:1234/Collection-App/">Home</a>
+        <a class="Genre" href="#">Genre</a>
+        <a class="Add-title" href="#">Add Movie</a>
+    
+</div>
+
+
+
+<?php
+
+
+
+$db = new PDO('mysql:host=db; dbname=movies', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$inputtedCaregory =
```

Comment:
You should remove this, it's causing an error

Reply:
line 29 has been amended